workflow/energy_rating_index.rb

```diff
@@ -29,10 +29,12 @@ def get_program_versions(hpxml_doc)
   end
   es_version = XMLHelper.get_value(hpxml_doc, '/HPXML/SoftwareInfo/extension/EnergyStarCalculation/Version', :string)
   iecc_version = XMLHelper.get_value(hpxml_doc, '/HPXML/SoftwareInfo/extension/IECCERICalculation/Version', :string)
+  zerh_version = XMLHelper.get_value(hpxml_doc, '/HPXML/SoftwareInfo/extension/ZERHCalculation/Version', :string)
 
   { [Constants.ERIVersions, 'ERICalculation/Version'] => eri_version,
     [ESConstants.AllVersions, 'EnergyStarCalculation/Version'] => es_version,
-    [IECCConstants.AllVersions, 'IECCERICalculation/Version'] => iecc_version }.each do |values, version|
+    [IECCConstants.AllVersions, 'IECCERICalculation/Version'] => iecc_version,
+    [ZERHConstants.Ver1, 'ZERHCalculation/Version'] => zerh_version }.each do |values, version|
```

Comment:
Seems like this should be `ZERHConstants.AllVersions` (based on the lines above).